### PR TITLE
Reusable Upgrade component for teams only features

### PR DESCRIPTION
**Pull request by CarinaWolli**
## What does this PR do?

Create a reusable component for the `Upgrade` badge that is shown for TEAMS only features (if user is not on TEAMS plan)

Component is used for:
- Video Recordings
- Disable branding in settings 
- SMS workflow actions

**Environment**: Staging(main branch)

## How should this be tested?

- Check if `Upgrade` badge is shown correctly when user does not have a team

### CODE REVIEW

**Comment by CarinaWolli** [comment]
@Udit-takkar could you test if everything is working as expected in the Video Recording dialog? I couldn't make it work to have recordings, so I couldn't fully test it

File: apps/web/pages/settings/my-account/appearance.tsx [hunk 1]
```diff
@@ -180,18 +182,18 @@ const AppearanceView = () => {
                   <p className="font-semibold ltr:mr-2 rtl:ml-2">
                     {t("disable_cal_branding", { appName: APP_NAME })}
                   </p>
-                  <Badge variant="gray">{t("pro")}</Badge>
+                  {!dataHasTeamPlan?.hasTeamPlan && <UpgradeTeamsBadge />}
```
Comment (PeerRich) [inline apps/web/pages/settings/my-account/appearance.tsx:185]: nice solution

File: packages/features/ee/video/ViewRecordingsDialog.tsx [hunk 2]
```diff
@@ -104,57 +109,48 @@ export const ViewRecordingsDialog = (props: IViewRecordingsDialog) => {
       <DialogContent>
         <DialogHeader title={t("recordings_title")} subtitle={subtitle} />
         <LicenseRequired>
-          {showUpgradeBanner && <UpgradeRecordingBanner />}
```
[the remark this answers is not in the document]
Comment (PeerRich) [reply packages/features/ee/video/ViewRecordingsDialog.tsx:107]: are you sure about this?

**Review by PeerRich — APPROVED** [review]
great work!

**Pull request merged**